Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -759,6 +759,14 @@
         "About": "The ThemeListener allows you to keep track of changes to the System Theme.",
         "Icon": "/Assets/Helpers.png",
         "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/helpers/ThemeListener.md"
+      },
```

Comment:
Kind of feel like this should be in `Extensions` as I'd expect that'd be where someone looking for a Markup Extension to go?